src/devices/DCMotor/DCMotor.csproj

```diff
@@ -9,6 +9,7 @@
     <Compile Include="DCMotor3Pin.cs" />
     <Compile Include="DCMotor2PinNoEnable.cs" />
     <Compile Include="DCMotor2PinWithBiDirectionalPin.cs" />
+    <Compile Include="DCMotorWithStartStop.cs" />
```

Comment:
Thanks for the fix. However,  wouldn't this better be changed to just a single Include="*.cs"? Seems to be what's used in most places. (If not we should check why the bindings differ in how the source is included)

Reply:
You're right, it's a better way 👍